bin/hive-config.sh

```diff
@@ -68,3 +68,7 @@ export HIVE_AUX_JARS_PATH=$HIVE_AUX_JARS_PATH
 
 # Default to use 256MB 
 export HADOOP_HEAPSIZE=${HADOOP_HEAPSIZE:-256}
+
+# Disable the JNDI. This feature has critical REC vulnerability.
```

Comment:
Small nitpick on a typo: "RCE" instead of "REC".

Reply:
Thanks for the review. 
Updated the comment